### ISSUE TITLE
Modify import paths when transpiling to CJS

### ISSUE THREAD

**Issue by christian-bromann** [egoist/tsup#666]
I'ld like to use `tsup` to transpile the [WebdriverIO project](https://github.com/webdriverio/webdriverio) into ESM and CJS exports. I see the following issue: given I a module with the following import:

```js
import moduleA from './moduleA.js
```
Now when I transpile the code into `file.js` (ESM) and `file.cjs` (CJS) the `file.cjs` will have the following transpiled code:

```js
var import_moduleA = require("./moduleA.js");
```
which points to the ESM version.

What is the recommended way to setup tsup to allow both exports.

**Comment by br14n-sol** [comment]
same issue :/

**Comment by 0x80** [comment]
This only occurs if you choose to preserve the file structure, right? Otherwise, your output will not contain relative imports I think. 
So maybe consider letting TSUP bundle things, as a workaround?

**Comment by br14n-sol** [comment]
> This only occurs if you choose to preserve the file structure, right? Otherwise, your output will not contain relative imports I think. So maybe consider letting TSUP bundle things, as a workaround?

Yes, it only happens when you want to preserve the file structure, in my case the entry was `./src/**/*.ts` and bundle set to false.

**Comment by br14n-sol** [comment]
As in my case it was not necessary to preserve the file structure I simply changed bundle to true and the entry to `./src/index.ts`.
